src/patternfly/components/Nav/nav.scss

```diff
@@ -338,6 +338,7 @@
   --pf-c-nav--c-menu__item--xl--PaddingRight: var(--pf-global--spacer--lg);
   --pf-c-nav--c-menu__item--xl--PaddingLeft: var(--pf-global--spacer--lg);
   --pf-c-nav--c-menu__item--FontSize: var(--pf-global--FontSize--sm);
+  --pf-c-nav--c-menu__item--Color: initial;
```

Comment:
this is going to be `canvastext`. Could you just change where this var is used to `--pf-c-menu__item--Color: var(--pf-c-nav--c-menu__item--Color, inherit);`?